### ISSUE TITLE
operation is finished and cannot be enqueued, while developing

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/instabug-reactnative#105]
Hey there,

when I reload the app, I sometimes get:

```
operation is finished and cannot be enqueued' was thrown while invoking startWithToken on target Instabug with params (
    d22dgbdawf5b2e7006babc7TESTe4175ce,
    2
)
callstack: (
	0   CoreFoundation                      0x000000011197212b __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x0000000110fd8f41 objc_exception_throw + 48
	2   CoreFoundation                      0x00000001119e7245 +[NSException raise:format:] + 197
	3   Foundation                          0x00000001109e5cdb __addOperations + 1099
	4   InstabugCore                        0x00000001108385fa -[IBGIntializationManager addOperation:] + 79
	5   Instabug                            0x000000011070126b +[Instabug startWithToken:invocationEvent:] + 342
	6   AcuityApp                           0x000000010b18f0a4 -[InstabugReactBridge startWithToken:invocationEvent:] + 100
	7   CoreFoundation                      0x00000001118f636c __invoking___ + 140
	8   CoreFoundation                      0x00000001118f6240 -[NSInvocation invoke] + 320
	9   CoreFoundation                      0x000000011190ec26 -[NSInvocation invokeWithTarget:] + 54
	10  AcuityApp                           0x000000010af2c38c -[RCTModuleMethod invokeWithBridge:module:arguments:] + 2796
	11  AcuityApp                           0x000000010afc4bd2 _ZN8facebook5reactL11invokeInnerEP9RCTBridgeP13RCTModuleDatajRKN5folly7dynamicE + 786
	12  AcuityApp                           0x000000010afc46ff _ZZN8facebook5react15RCTNativeModule6invokeEjON5folly7dynamicEiENK3$_0clEv + 127
	13  AcuityApp                           0x000000010afc4679 ___ZN8facebook5react15RCTNativeModule6invokeEjON5folly7dynamicEi_block_invoke + 25
	14  libdispatch.dylib                   0x0000000113f792f7 _dispatch_call_block_and_release + 12
	15  libdispatch.dylib                   0x0000000113f7a33d _dispatch_client_callout + 8
	16  libdispatch.dylib                   0x0000000113f855f9 _dispatch_main_queue_callback_4CF + 628
	17  CoreFoundation                      0x0000000111934e39 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	18  CoreFoundation                      0x00000001118f9462 __CFRunLoopRun + 2402
	19  CoreFoundation                      0x00000001118f8889 CFRunLoopRunSpecific + 409
	20  GraphicsServices                    0x00000001144949c6 GSEventRunModal + 62
	21  UIKit                               0x000000010e4745d6 UIApplicationMain + 159
	22  AcuityApp                           0x000000010acbcedf main + 111
	23  libdyld.dylib                       0x0000000113ff6d81 start + 1
	24  ???                                 0x0000000000000001 0x0 + 1

```

and the app crashes.

That's on iOS with:
instabug-reactnative 2.1.1
react-native: 0.51.0

**Comment by Kmohamed** [comment]
Hello @SudoPlz,
The only cause of this crash that you are calling startWithToken twice. could you double check this please.

**Comment by SudoPlz** [comment]
@Kmohamed that's not true, I only call it once, but when the app reloads (on Debug mode), it's probably called again when the app starts (which is normal). I believe the module should not crash but fail gracefully on dev reloads.

**Comment by Kmohamed** [comment]
@sudoPlz sorry for that 😐 
we will release a fix today.

**Comment by SudoPlz** [comment]
Perfect, I'll be waiting for the release my friend.
:)

**Comment by Kmohamed** [comment]
Thanks my friend😊😊

**Comment by SudoPlz** [comment]
Ok another friend noticed that here: https://github.com/Instabug/instabug-reactnative/issues/106

Are there any quick fixes for now? This really slows our development process.
Thank you.

**Comment by salmatarzi** [comment]
@SudoPlz The fix for this issue is now released, kindly upgrade to v2.1.2

**Comment by SudoPlz** [comment]
You are THE best!

Thank you!!!
